### PR TITLE
Add request parameters to Python models

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

**Review by tanmayv25 — DISMISSED** [review]

**Review by krishung5 — APPROVED** [review]

**Review by GuanLuo — COMMENTED** [review]
So the Python model read the parameters as a JSON string?

**Comment by Tabrizian** [comment]
@GuanLuo yes, that's correct.

File: src/python_be.cc [hunk 1]
```diff
@@ -356,6 +356,39 @@ ModelInstanceState::SaveRequestsToSharedMemory(
...
+            TRITONSERVER_ERROR_INVALID_ARG,
+            (std::string("Unsupported parameter type for parameter '") + name +
+             "'.")
+                .c_str());
+      }
+    }
+
+    triton::common::TritonJson::WriteBuffer buffer;
+    RETURN_IF_ERROR(parameters_json.Write(&buffer));
+    const auto& parameters_string = buffer.Contents();
```
Comment (GuanLuo) [inline src/python_be.cc:390]: Document the JSON format of parameters that the Python model will receive.

Comment (Tabrizian) [reply src/python_be.cc:390]: Added a documentation.

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**